app/main/routes.py

```diff
@@ -0,0 +1,71 @@
+from flask import flash, json, make_response, redirect, render_template, request
+from flask_wtf.csrf import CSRFError
+from werkzeug.exceptions import HTTPException
+
+from app.main import bp
+from app.main.forms import CookiesForm
+
+
+@bp.route("/", methods=["GET", "POST"])
```

Comment:
I don't think we need a `POST` method on this route?

Reply:
We will need a post method once we begin connecting to the backend, so I put one in as a placeholder. I can remove for now though.